File: C/Lista_03(Estrutura_Repetitiva)/Experiencias.c
#include <stdio.h>

int main () {

    int casos;

    printf("Quantos casos de teste serao digitados? ");
    scanf("%d", &casos);

    int quantidade;
    char tipo;
    int coelho=0, rato=0, sapo=0, total=0;

    for(int i=0;i<casos;i++){

        printf("Quantidade de cobaias: ");
        scanf("%d", &quantidade);

        printf("Tipo de cobaia: ");
        scanf(" %c", &tipo);

        total += quantidade;

        if (tipo=='C'){
            coelho += quantidade;
        }
        else if (tipo=='R'){
            rato += quantidade;
        }
        else if (tipo=='S'){
            sapo += quantidade;
        }
    }

    printf("\nRelatorio final:\n");

    printf("Total: %d cobaias\n", total);
    printf("Total de coelhos: %d\n", coelho);
    printf("Total de ratos: %d\n", rato);
    printf("Total de sapos: %d\n", sapo);

    double perC = (double) coelho*100/total;
    double perR = (double) rato*100/total;
    double perS = (double) sapo*100/total;

    printf("Percentual de coelhos: %.2lf\n", perC);
    printf("Percentual de ratos: %.2lf\n", perR);
    printf("Percentual de sapos: %.2lf\n", perS);

    return 0;
}

File: Python/Lista_03(Estrutura_Repetitiva)/Experiencias.py
n = int(input("Quantos casos de teste serao digitados? "))

coelhos = 0
ratos = 0
sapos = 0
total = 0

for i in range(0,n):
    cobaias = int(input("Quantidade de cobaias: "))
    tipo = input("Tipo de cobaia: ")
    if tipo=='C':
        coelhos+=cobaias
    elif tipo=='R':
        ratos+=cobaias
    elif tipo=='S':
        sapos+=cobaias
    total+=cobaias

print("\nRELATORIO FINAL:")
print(f"Total: {total} cobaias")
print(f"Total de coelhos: {coelhos}")
print(f"Total de ratos: {ratos}")
print(f"Total de sapos: {sapos}")

perC = coelhos/total*100
perR = ratos/total*100
perS = sapos/total*100

print(f"Percentual de coelhos: {perC:.2f}")
print(f"Percentual de ratos: {perR:.2f}")
print(f"Percentual de sapos: {perS:.2f}")